sprinklers/base.py

```diff
@@ -1,3 +1,4 @@
+import app_settings
```

Comment:
Use relative imports for clarity:
```suggestion
from . import app_settings
```